test/extensions/filters/common/expr/evaluator_fuzz_test.cc

```diff
@@ -0,0 +1,74 @@
+#include "common/network/utility.h"
+
+#include "extensions/filters/common/expr/evaluator.h"
+
+#include "test/common/stream_info/test_util.h"
+#include "test/extensions/filters/common/expr/evaluator_fuzz.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/fuzz/utility.h"
+#include "test/test_common/network_utility.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace Expr {
+namespace {
+
+// Set stream_info using information from input.
+// Duplicates most of fromStreamInfo, except is more lightweight without the use of mocks.
```

Comment:
Should this be a general fuzzing util in `test/fuzz/utility.cc` or somewhere like that?

Reply:
Done. I added a TODO to make it faster. I originally didn't have it use the existing method because of the mock usage, but that should be fixed across the board.